blueprints/networking/filtering-proxy-psc/main.tf

```diff
@@ -0,0 +1,300 @@
+  target_service        = module.squid-ilb.forwarding_rule_self_link
+  consumer_accept_lists {
+    project_id_or_num = module.project.project_id
+    connection_limit  = 10
+  }
+}
+
+###############################################################################
+#                               Squid resources                               #
+###############################################################################
+
+module "service-account-squid" {
+  source     = "../../../modules/iam-service-account"
+  project_id = module.project.project_id
+  name       = "svc-squid"
+  iam_project_roles = {
+    (module.project.project_id) = [
+      "roles/logging.logWriter",
+      "roles/monitoring.metricWriter",
+    ]
+  }
+}
+
+module "cos-squid" {
+  source  = "../../../modules/cloud-config-container/squid"
+  allow   = var.allowed_domains
+  clients = [var.cidrs.psc]
+}
+
+module "squid-vm" {
+  source          = "../../../modules/compute-vm"
+  project_id      = module.project.project_id
+  zone            = "${var.region}-b"
+  name            = "squid-vm"
+  instance_type   = "e2-medium"
+  create_template = true
+  network_interfaces = [{
+    network    = module.vpc.self_link
+    subnetwork = module.vpc.subnet_self_links["${var.region}/proxy"]
+  }]
+  boot_disk = {
+    image = "cos-cloud/cos-stable"
+  }
+  service_account        = module.service-account-squid.email
+  service_account_scopes = ["https://www.googleapis.com/auth/cloud-platform"]
+  metadata = {
+    user-data = module.cos-squid.cloud_config
+  }
+}
+
+module "squid-mig" {
+  source            = "../../../modules/compute-mig"
+  project_id        = module.project.project_id
+  location          = "${var.region}-b"
+  name              = "squid-mig"
+  instance_template = module.squid-vm.template.self_link
+  target_size       = 1
+  auto_healing_policies = {
+    initial_delay_sec = 60
+  }
+  autoscaler_config = {
+    max_replicas    = 10
+    min_replicas    = 1
+    cooldown_period = 30
+    scaling_signals = {
+      cpu_utilization = {
+        target = 0.65
+      }
+    }
+  }
+  health_check_config = {
+    enable_logging = true
+    tcp = {
+      port = 3128
+    }
+  }
+  update_policy = {
+    minimal_action = "REPLACE"
+    type           = "PROACTIVE"
+    max_surge = {
+      fixed = 3
+    }
+    min_ready_sec = 60
+  }
+}
+
+module "squid-ilb" {
+  source        = "../../../modules/net-ilb"
+  project_id    = module.project.project_id
+  region        = var.region
+  name          = "squid-ilb"
+  ports         = [3128]
+  service_label = "squid-ilb"
+  vpc_config = {
+    network    = module.vpc.self_link
+    subnetwork = module.vpc.subnet_self_links["${var.region}/proxy"]
+  }
+  backends = [{
+    group = module.squid-mig.group_manager.instance_group
+  }]
+  health_check_config = {
+    enable_logging = true
+    tcp = {
+      port = 3128
+    }
+  }
+}
+
+###############################################################################
```

Comment:
ohhh, consumer is in here. can we split it into separate .tf file?

Reply:
Done - moved all consumer resources to consumer.tf.